src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, AreaCalculator {
```

Comment:
This task doesn't require creating a new Figure. It challenges your knowledge of working with abstraction.